client/state/posts/actions.js

```diff
@@ -40,7 +40,7 @@ export function receivePosts( posts ) {
 /**
  * Triggers a network request to fetch posts for the specified site and query.
  *
- * @param  {Number}   siteId Site ID
+ * @param  {?Number}  siteId Site ID
```

Comment:
In looking it up again myself, I notice there's also a `!` prefix to indicate that a value should _not_ be `null`, which seems to imply that an indicator is optional, and without one could reasonably return `null`. Still nice to have, I think.